production label (#498)
neo4j/docs-aura: modules/ROOT/pages/managing-instances/instance-actions.adoc

====
There is no way to recover data from a deleted Aura instance.
====

> == Mark an instance as production

Reviewer: ```suggestion
== Mark an instance as `production`
```

label:AuraDB-Virtual-Dedicated-Cloud[]
label:AuraDB-Business-Critical[]

Admins can mark a database instance as `production` in AuraDB Virtual Dedicated Cloud and AuraDB Business Critical.

> When Neo4j updates Aura Database Versions, instances marked as `production` are last to have the Aura Database Version updated. Because updates are applied to these database instances after other database instances recieve updates, you can monitor any potential impact on less critical databases first.

Reviewer: ```suggestion
When Neo4j updates Aura Database versions, instances marked as `production` are last to have the Aura Database version updated. 
Because updates are applied to these database instances after other database instances receive updates, you can monitor any potential impact on less critical instances first.
```

> After marking the instance as `production` the label is applied immediately, and all instance actions (such as pause or clone) will be temporarily unavailable while the instance is set to `production` on the Neo4j side. This should only take a few moments.

Reviewer: ```suggestion
After marking the instance as `production`, the label is applied immediately, and all instance actions (such as pause or clone) are temporarily unavailable while the instance is set to `production` on the Neo4j side. 
```

Reviewer: What do you mean with _on the Neo4j side_?

Author: Whilst Aura updates the instance to `production` in the Aura backend.

> To mark an instance as `production` click the more actions (three dots) on the instance card, then "Mark as production".

Reviewer: ```suggestion
Use the more menu (three dots) on the instance card to mark it as `production`.
```
